### PR TITLE
Issue 104

### PR DESCRIPTION
**Pull request by jblouse**
I added the skeleton of the work and am still working on the ECS code. I moved some hard coded strings to constants as well. I got tired of restringing so to speak. Let me know if you object for any reason. I figured out why my last build didn't fail as well locally. I was building in gogland and somehow it runs differently even though I used the same command. I will be more careful the next time. When I ran it from the terminal it failed as expected. In the spirit of CI, I wanted to get you a sample and also check to make sure I am doing what you expected (putting the right pieces in the right places, code style, etc.). This code doesn't do much yet but I have the skeleton unless I missed something.

### CODE REVIEW

File: cli/services.go [hunk 1]
```diff
@@ -90,55 +90,103 @@ func newServicesDeployCommand(ctx *common.Context) *cli.Command {
...
+
+	if argLength == common.Zero || len(strings.TrimSpace(environmentName)) == common.Zero {
+		cli.ShowCommandHelp(ctx, common.ExeCmd)
+		return errors.New(common.NoEnvValidation)
+	}
+	if argLength == common.ExeArgsSvcIndex {
+		cli.ShowCommandHelp(ctx, common.ExeCmd)
+		return errors.New(common.NoSvcValidation)
+	}
+	if len(strings.TrimSpace(ctx.Args().Get(common.ExeArgsSvcIndex))) == common.Zero {
```
Comment (cplee) [inline cli/services.go]: serviceName should be optional, passed as a `-s <service>` flag.  The default (if not provided) will be to pull the serviceName from the `mu.yml`, if file exists.

File: workflows/service_executor.go [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+	"github.com/stelligent/mu/common"
+)
+
+// NewServiceExecutor create a new workflow for executing a command in an environment
+func NewServiceExecutor(ctx *common.Context, environmentName string, service string, command string) Executor {
+
+	workflow := new(environmentWorkflow)
+
+	return newWorkflow(
+		workflow.serviceTaskExecutor(environmentName, service, command, ctx.TaskManager),
```
Comment (cplee) [inline workflows/service_executor.go]: need another step in workflow to lookup the serviceName from `mu.yml` if none is provided.

**Comment by jblouse** [comment]
Sounds good. I will update soon.

**Comment by jblouse** [comment]
I believe this is good to go. Let me know if any changes are needed or if I didn't quite capture it the way you expected.

**Pull request merged**